labs/playground1/Makefile

```diff
@@ -56,6 +56,15 @@ pkg-extension-authenticator-canner: ../../node_modules
 	rm -rf ./labs/playground1/node_modules/@vulcan-sql/extension-authenticator-canner; \
 	cp -R ./dist/packages/extension-authenticator-canner ./labs/playground1/node_modules/@vulcan-sql
 
+pkg-extension-driver-clickhouse: ../../node_modules
```

Comment:
is this necessary? It seems like the vulcan.yaml didn't use the clickhouse extension